src/main/java/org/rpgl/function/InvokeEvent.java

```diff
@@ -0,0 +1,26 @@
+package org.rpgl.function;
+
+import org.rpgl.core.RPGLContext;
+import org.rpgl.core.RPGLFactory;
+import org.rpgl.core.RPGLObject;
+import org.rpgl.json.JsonObject;
+import org.rpgl.subevent.Subevent;
+
+public class InvokeEvent extends Function {
+
+    public InvokeEvent() {
+        super("invoke_event");
+    }
+
+    @Override
+    public void execute(RPGLObject effectSource, RPGLObject effectTarget, Subevent subevent,
+                        JsonObject functionJson, RPGLContext context) throws Exception {
+        super.verifyFunction(functionJson);
+        effectTarget.invokeEvent(
+                new RPGLObject[] {}, // TODO this needs to be supplied somehow
```

Comment:
Since this is not fully supported, it should be commented out in Functions.FUNCTION for the time being